From b1a25e3f746fc220f50afb484e4bb19d9fd33749 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:33 -0400
Subject: [usb] core/devio: Spaces to tabs for proc_control_compat()

Message-id: <1410887463-20674-82-git-send-email-dzickus@redhat.com>
Patchwork-id: 95206
O-Subject: [RHEL7 PATCH 081/591] drivers: usb: core: devio.c: Spaces to tabs for proc_control_compat()
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 06793f2d0cf9daabeeb5b024d4bf7082dcc71505
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Mon Oct 14 21:46:37 2013 +0200

    drivers: usb: core: devio.c: Spaces to tabs for proc_control_compat()

    Replaced spaces by tabs for proc_control_compat() function.

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/devio.c b/drivers/usb/core/devio.c
index c697463..967152a 100644
--- a/drivers/usb/core/devio.c
+++ b/drivers/usb/core/devio.c
@@ -1647,19 +1647,19 @@ static int proc_control_compat(struct dev_state *ps,
 static int proc_bulk_compat(struct dev_state *ps,
    struct usbdevfs_bulktransfer32 __user *p32)
 {
-        struct usbdevfs_bulktransfer __user *p;
-        compat_uint_t n;
-        compat_caddr_t addr;
+ struct usbdevfs_bulktransfer __user *p;
+ compat_uint_t n;
+ compat_caddr_t addr;
 
-        p = compat_alloc_user_space(sizeof(*p));
+ p = compat_alloc_user_space(sizeof(*p));
 
-        if (get_user(n, &p32->ep) || put_user(n, &p->ep) ||
-            get_user(n, &p32->len) || put_user(n, &p->len) ||
-            get_user(n, &p32->timeout) || put_user(n, &p->timeout) ||
-            get_user(addr, &p32->data) || put_user(compat_ptr(addr), &p->data))
-                return -EFAULT;
+ if (get_user(n, &p32->ep) || put_user(n, &p->ep) ||
+     get_user(n, &p32->len) || put_user(n, &p->len) ||
+     get_user(n, &p32->timeout) || put_user(n, &p->timeout) ||
+     get_user(addr, &p32->data) || put_user(compat_ptr(addr), &p->data))
+  return -EFAULT;
 
-        return proc_bulk(ps, p);
+ return proc_bulk(ps, p);
 }
 static int proc_disconnectsignal_compat(struct dev_state *ps, void __user *arg)
 {
-- 
1.7.1